Branch-SDK/src/io/branch/referral/Branch.java

```diff
@@ -3619,7 +3619,7 @@ public void run() {
 	 * @see BranchError	
 	 */
 	public interface BranchListResponseListener {
-		public void onReceivingResponse(JSONArray list, BranchError error);
+		void onReceivingResponse(JSONArray list, BranchError error);
```

Comment:
Correct, and sounds good. If anything, the bosses know where I am - just pick me up if anyone needs offline discussion.